packages/oc-template-handlebars-compiler/test/__snapshots__/compile.test.js.snap

```diff
@@ -34,11 +34,131 @@ Object {
 
 exports[`Should correctly compile the oc component 2`] = `
 Array [
-  "../../../mocks/handlebars-component/__package/assets/css/style.css",
-  "../../../mocks/handlebars-component/__package/assets/img/test.png",
-  "../../../mocks/handlebars-component/__package/assets/js/lib.js",
-  "../../../mocks/handlebars-component/__package/package.json",
-  "../../../mocks/handlebars-component/__package/server.js",
-  "../../../mocks/handlebars-component/__package/template.js",
+  "../../../mocks/handlebars-component/_compile-tests-package/assets/css/style.css",
```

Comment:
I decided to provide more meaningful names to package dest folders, and each test has its own so that running them asynchronously should be fine